opennms-full-assembly/pom.xml

```diff
@@ -301,6 +301,33 @@
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
+        <groupId>net.nicoulaj.maven.plugins</groupId>
+        <artifactId>checksum-maven-plugin</artifactId>
+        <version>1.5</version>
+        <executions>
+          <execution>
+            <phase>prepare-package</phase>
+            <goals><goal>files</goal></goals>
+          </execution>
+        </executions>
+        <configuration>
+          <algorithms><algorithm>SHA-1</algorithm></algorithms>
+          <fileSets>
+            <fileSet>
+              <directory>${project.build.directory}/features-repo</directory>
+              <includes>
+                <include>**/*.cfg</include>
+                <include>**/*.jar</include>
+                <include>**/*.pom</include>
+                <include>**/*.war</include>
+                <include>**/*.xml</include>
+              </includes>
+            </fileSet>
+          </fileSets>
+        </configuration>
+      </plugin>
+      <!--
```

Comment:
If we need to disable the `maven-clean-plugin` here I'd suggest adding a note as to why, or removing the block completely.